src/main/webapp/teacher/controlGameInstance/controlGameInstance.js

```diff
@@ -479,9 +485,7 @@ function addQuestionAnswerToHistoryUI({ questionId, answerId, answer } = {}) {
   answerInQuestionStatsDivElement.innerText = answer.title + ' with ' + answer.numberAnswers + ' answers.';
 
   if (answer.correct) {
-    questionAnswerDivElement.innerText = 'Correctly answered, chose: "' + answer.chosen + '"';
-  } else {
-    questionAnswerDivElement.innerText = 'Incorrect answer, chose: "' + answer.chosen + '"';
```

Comment:
removed this on purpose?

Reply:
Yep that was there by mistake, was accessing nonexistent info and changed it to display the actual answer info.